packages/block-library/src/image/image.js

```diff
@@ -193,6 +196,24 @@ export default function Image( {
 	}
 
 	function onSetHref( props ) {
+		if (
+			inheritedAttributes?.linkDestination ||
+			inheritedAttributes?.linkTarget
+		) {
+			setAttributes( {
+				inheritedAttributes: {
+					...inheritedAttributes,
+					linkDestination:
+						props.linkDestination || props.href
+							? false
+							: inheritedAttributes.linkDestination,
+					linkTarget:
+						props.linkTarget || props.linkTarget !== linkTarget
+							? false
+							: inheritedAttributes.linkTarget,
+				},
+			} );
+		}
```

Comment:
What are  these changes need for?

Reply:
They were needed when the child images were handling the updating of change in values from the gallery and I missed removing these changes when I pushed this functionality bak up to the gallery yesterday. Tidied up now.